sirepo/auth/__init__.py

```diff
@@ -48,6 +49,11 @@
 
 _GUEST_USER_DISPLAY_NAME = 'Guest User'
 
+_PAYMENT_PLAN_BASIC = 'basic'
+_PAYMENT_PLAN_ENTERPRISE = 'enterprise'
```

Comment:
I think it is reasonable to document:

```py
_PAYMENT_PLAN_ENTERPRISE = ROLE_PAYMENT_PLAN_ENTERPRISE
_PAYMENT_PLAN_PREMIUM = ROLE_PAYMENT_PLAN_PREMIUM
```

Reply:
https://github.com/radiasoft/sirepo/pull/2711/commits/05bde41c8e65d6cbc52101463c4a1a6418c2356b